client/src/components/Feed.jsx

```diff
@@ -10,8 +10,8 @@ class Feed extends React.Component {
 
   render() {
 
-    let hasTransactions = this.props.transactions.items && this.props.transactions.items.length > 0;
-    let showMoreButton = Boolean(this.props.transactions.nextPageTransactionId);
+    let hasTransactions = this.props.transactions && this.props.transactions.items && this.props.transactions.items.length > 0;
```

Comment:
Unclear why these changes were neccessary.

Reply:
They weren't - I thought they made the code clearer.

Reply:
Also - they protect the app in case no data is passed. if there are no transactions, I don't want the string "this.props.transactions.nextPageTransactionId" to even attempt to evaluate.